src/components/ChannelList/__tests__/ChannelList.test.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable sonarjs/no-unused-collection */
+import React from 'react';
+import axios from 'axios';
+import { cleanup, render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { Chat } from '../../Chat';
+import ChannelList from '../ChannelList';
+
+import {
+  getTestClient,
+  useMockedApis,
+  queryChannelsApi,
+  generateMessage,
+  generateChannel,
+  generateUser,
+  getOrCreateChannelApi,
+  dispatchMessageNewEvent,
+  dispatchNotificationAddedToChannelEvent,
+  getTestClientWithUser,
+  erroredGetApi,
+} from '__mocks__';
+
+// eslint-disable-next-line no-undef
```

Comment:
Not sure if it would be on the scope of this PR, but we can add jest globals to eslint using https://github.com/jest-community/eslint-plugin-jest.

Reply:
agree #230

Reply:
We can change the name? Any suggestions? How about `__mock-builders__`? Since we only export functions from this directory and not mock objects.